app/models/host/managed.rb

```diff
@@ -271,8 +271,12 @@ def built(installed = true)
     self.installed_at = Time.now.utc if installed
 
     if self.save
-      recipients = owner ? owner.recipients_for(:host_built) : []
-      MailNotification[:host_built].deliver(self, :users => recipients) if recipients.present?
+
+      if installed
```

Comment:
How about extracting this to a method `send_built_notification`? It would make this method a bit cleaner and we could test it in isolation.

```ruby
if self.save
  send_built_notification if installed
  true
else
```